common-pg/src/main/scala/com/socrata/pg/soql/SqlFunctions.scala

```diff
@@ -64,7 +64,7 @@ object SqlFunctions {
     // http://beta.dev.socrata.com/docs/datatypes/numeric.html
     UnaryPlus -> formatCall("%s"),
     UnaryMinus -> formatCall("-%s") _,
-    Magnitude -> formatCall("floor(log(%s, abs(%s) * %s + 1))", Some(Seq(1,0,1,0))),
```

Comment:
So we had the open question of if we can make 0 map to 0 with a case statement without impacting performance.  Did you try that?